src/libraries/System.Private.CoreLib/src/System/Text/ASCIIUtility.cs

```diff
@@ -42,6 +42,28 @@ private static bool AllCharsInUInt64AreAscii(ulong value)
             return (value & ~0x007F007F_007F007Ful) == 0;
         }
 
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public static int GetIndexOfFirstNonAsciiByteInLane(Vector128<byte> value, Vector128<byte> bitmask)
```

Comment:
Should this be public?

Reply:
Ah, oversight on my part. Fixed now. Thanks!